apps/ideas/views.py

```diff
@@ -83,7 +75,6 @@ def render_next_step(self, form, **kwargs):
         else:
             return super().render_next_step(form, **kwargs)
 
-    def done(self, form_list, **kwargs):
```

Comment:
When I try to save the form, I get a NotImplementedError, because the done method is missing.